### DISCUSSION TITLE
Installation updates

### DISCUSSION THREAD

**Discussion by jacobhroutzong** [Homebrew/discussions#2570] [Getting started]
I'm in a remote area where we get about 300 kb/s. I knew home-brew would take an hour or so to download. However, in the terminal Homebrew has said, "downloading command line tools for Xcode" for a while now. How do I check for a status update?

**Comment by SMillerDev** [comment]
Unfortunately that install is not something we control. If you want a clearer status indicator you could use `xcode-select --install` before installing brew.

**Reply by jacobhroutzong** [reply]
The install failed overnight during the aforementioned process. I'll try the Xcode-select --install.
